mbuild/formats/hoomd_snapshot.py

```diff
@@ -104,11 +103,11 @@ def to_hoomdsnapshot(
     structure : parmed.Structure
         ParmEd Structure object
     ref_distance : float, optional, default=1.0
-        Reference distance for conversion to reduced units
+        Reference distance for unit conversion (from Angstrom)
     ref_mass : float, optional, default=1.0
-        Reference mass for conversion to reduced units
+        Reference mass for unit conversion (from Dalton)
     ref_energy : float, optional, default=1.0
-        Reference energy for conversion to reduced units
+        Reference energy for unit conversion (from kcal/mol)
```

Comment:
```suggestion
        Reference energy for unit conversion ((desired units) / (kcal/mol))
```
Possible cleaner way to write the conversion, it's a more mathematical way to show it in my opinion. I'll defer to your judgement though. You could also use `target units` instead of `desired units`.

Reply:
I like this idea, but I think it would be opposite `(kcal/mol) / (desired units)` since it is dividing by the reference values.

Reply:
Agreed! Good idea.